index.js

```diff
@@ -170,10 +170,12 @@ export default class ModalSelector extends React.Component {
     }
 
     onChange = (item) => {
+        this.props.onChange(item);
+        /* this doesn't seem to play nice, the setState below doesn't propagate to the onDismiss state (changeItem) is undefined in the onDismiss function
         if (Platform.OS === 'android' || (Modal.propTypes !== undefined && !Modal.propTypes.onDismiss)) { // don't know if this will work for previous version, please check!
             // RN >= 0.50 on iOS comes with the onDismiss prop for Modal which solves RN issue #10471
             this.props.onChange(item);
-        }
+        }*/
```

Comment:
Please delete the unused code instead of commenting it out.  It will be available in the file history.

Reply:
Sure thing.